assemblies/assembly_bpmn-reference/dm/master-docinfo.xml

```diff
@@ -3,6 +3,6 @@
 <subtitle>
 </subtitle>
 <abstract>
-	<para>A BPMN2 reference guide for the {PRODUCT} {PRODUCT_VERSION} process designer.</para>
+	<para>This document is a BPMN2 reference guide for the {PRODUCT} {PRODUCT_VERSION} legacy and new process designers.</para>
```

Comment:
Just a general comment that this doc has a lot of passive, most of which is called for because of the nature of describing things, but some can be avoided. Not saying to worry about it or do much, just pointing it out for future ref.

Also, this content is another clear candidate for single sourcing :) Could even just go in exactly as is as, might not even need to bring in or incorporate things from community. We should chat sometime.